CardPhunTests/CardTests/CardSetTest.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Linq;
+using CardPhun;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace CardPhunTests.CardTests
+{
+    [TestClass]
+    public class CardSetTest
+    {
+        [TestMethod]
+        public void AddToSetCountTest()
+        {
+            var card = new TestCard(1, Znak.CLUBS);
+            var cardSet = new TestCardSet();
+            cardSet.AddToSet(card);
+            Assert.AreEqual(cardSet.Count, 1);
+        }
+
+        [TestMethod]
+        public void AddToSetNumberTest()
+        {
+            var card = new TestCard(1, Znak.CLUBS);
+            var cardSet = new TestCardSet();
+            cardSet.AddToSet(card);
+            Assert.AreEqual(cardSet.GetSumOfCards(), 1);//da li sam ovim proverio getsumofcards, ili to moram posebno kada dodjem do toga?
+        }
+
+        [TestMethod]
+        public void SeeCardTest()
+        {
+            var card = new TestCard(1, Znak.CLUBS);
+            var cardSet = new TestCardSet();
+            cardSet.AddToSet(card);
+            Assert.AreEqual(cardSet.SeeCard(0).ToString(), "1 CLUBS");
+        }
+
+        [TestMethod]
+        public void ShuffleTest()
+        {
+            var cardSet = new TestCardSet();
+            var shuffledCardSet = new TestCardSet();
+            for (var i = 0; i < 10; i++)
+            {
+                var card = new TestCard(i+1, Znak.CLUBS);
+                cardSet.AddToSet(card);
+                shuffledCardSet.AddToSet(card);
+            }
+            shuffledCardSet.Shuffle();
+            Assert.AreNotEqual(shuffledCardSet, cardSet);
+
```

Comment:
e vidis, secam se sta si mi pricao, ali jos pre nego sto probam sa 99% siguran da necu uspeti, pa bi to bilo dobro da pogledamo zajedno...